components/eam/src/physics/cam/cospsimulator_intr.F90

```diff
@@ -1045,12 +1061,24 @@ subroutine cospsimulator_intr_init()
        ! float clmodis ( time, plev, tau, loc )
        call addfld ('CLMODIS',(/'cosp_tau_modis','cosp_prs      '/),'A','%','MODIS Cloud Area Fraction',            &
             flag_xyfill=.true., fill_value=R_UNDEF)
+       ! float clliqmodis ( time, plev, tau, loc )
+       call addfld ('CLLIQMODIS',(/'cosp_tau_modis','cosp_prs      '/),'A','%','MODIS Cloud Area Fraction',            &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! float clicemodis ( time, plev, tau, loc )
+       call addfld ('CLICEMODIS',(/'cosp_tau_modis','cosp_prs      '/),'A','%','MODIS Cloud Area Fraction',            &
+            flag_xyfill=.true., fill_value=R_UNDEF)
        ! float clrimodis ( time, plev, tau, loc )
        call addfld ('CLRIMODIS',(/'cosp_tau_modis','cosp_reffice  '/),'A','%','MODIS Cloud Area Fraction',            &
             flag_xyfill=.true., fill_value=R_UNDEF)
        ! float clrlmodis ( time, plev, tau, loc )
        call addfld ('CLRLMODIS',(/'cosp_tau_modis','cosp_reffliq  '/),'A','%','MODIS Cloud Area Fraction',            &
             flag_xyfill=.true., fill_value=R_UNDEF)
+       ! float iwprimodis ( time, reffice, iwp, loc )
+       call addfld ('IWPRIMODIS',(/'cosp_iwp_modis','cosp_reffice  '/),'A','%','MODIS Cloud Area Fraction',            &
+            flag_xyfill=.true., fill_value=R_UNDEF)
+       ! float lwprlmodis ( time, reffliq, lwp, loc )
+       call addfld ('LWPRLMODIS',(/'cosp_lwp_modis','cosp_reffliq  '/),'A','%','MODIS Cloud Area Fraction',            &
+            flag_xyfill=.true., fill_value=R_UNDEF)
```

Comment:
@zyuying The longnames for all the above fields are called "MODIS Cloud Area Fraction". Useful to make them distinguishable.

Reply:
@wlin7 I have changed the longnames for all the joint-histogram variables. Thanks.